compiler/src/dotty/tools/dotc/util/SourcePosition.scala

```diff
@@ -23,7 +23,7 @@ extends interfaces.SourcePosition with Showable {
   def point: Int = span.point
 
   /** The line of the position, starting at 0 */
```

Comment:
Doc comment can be removed since the interface already has one which is more accurate (same with column)